customer_device_manager/security/ir.model.access.csv

```diff
@@ -0,0 +1,9 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+acl_device_assignment_portal,access.device_assignment.portal,model_customer_device_manager_device_assignment,base.group_portal,1,0,0,0
+acl_device_assignment_history_portal,access.device_assignment_history.portal,model_customer_device_manager_device_assignment_history,base.group_portal,1,0,0,0
+acl_device_assignment_assigner,access.device_assignment.assigner,model_customer_device_manager_device_assignment,customer_device_manager.group_customer_device_assigner,1,1,0,0
+acl_device_assignment_internal,access.device_assignment.internal,model_customer_device_manager_device_assignment,base.group_user,1,0,0,0
+acl_device_assignment_history_internal,access.device_assignment_history.internal,model_customer_device_manager_device_assignment_history,base.group_user,1,0,0,0
+acl_device_assignment_manager,access.device_assignment.manager,model_customer_device_manager_device_assignment,customer_device_manager.group_user_manager,1,1,1,0
+acl_device_assignment_history_manager,access.device_assignment_history.manager,model_customer_device_manager_device_assignment_history,customer_device_manager.group_user_manager,1,0,0,0
```

Comment:
On fait le choix ici que personne en interne ne peut modifier l'historique, je pense que c'est pas une bonne idée, en cas d'erreur c'est l'IT qui devra corriger... Ajouter tous les droits sur l'historique au group_user_manager me semble nécessaire, qu'en dis-tu ?

Reply:
Tout à fait d'accord, +1 et fait !